csv_util.cc

```diff
@@ -1215,11 +1215,11 @@ xcsv_parse_val(const char* s, Waypoint* wpt, const field_map_t* fmp,
     wpt->SetCreationTime(sscanftime(s, fmp->printfc, 1));
     break;
   case XT_LOCAL_TIME:
-    if (getenv("GPSBABEL_FREEZE_TIME")) {
+    if (ugetenv("GPSBABEL_FREEZE_TIME").isNull()) {
```

Comment:
I flipped the if/else clauses rather than doing a double negative "!...isNull()".